eskipfile/file.go

```diff
@@ -45,9 +45,16 @@ func Open(path string) (*Client, error) {
 	return &Client{routes}, nil
 }
 
+func (c Client) LoadAndParseAll() (routeInfos []*eskip.RouteInfo, err error) {
+	for _, route := range c.routes {
+		routeInfos = append(routeInfos, &eskip.RouteInfo{*route, nil})
```

Comment:
I don't get the problem. In order for a client to load and parse the routes, it need to implement the correct interface, isn't it?